packages/common/src/downloads/helpers/canCreateExportItem.ts

```diff
@@ -0,0 +1,20 @@
+import { IArcGISContext } from "../../ArcGISContext";
+import { IHubEditableContent } from "../../core/types/IHubEditableContent";
+
+/**
+ * @private
+ * Determines if the current user can create an export item for the given entity.
+ *
+ * NOTE: This function is a placeholder. Various permissions and logic branches are not yet implemented.
+ *
+ * @param _entity
+ * @param _context
+ * @returns
+ */
+export function canCreateExportItem(
+  _entity: IHubEditableContent,
+  _context: IArcGISContext
+) {
+  // TODO: port over logic from the download-service
```

Comment:
confirming that you didn't intend to do this in this PR

Reply:
That is correct. I thought about deleting all the code for the export item flow (including this helper), but since I had gone so far down the path, I figured it would be better to check the work in progress stuff in and flesh it out in a follow on PR.